pyproject.toml

```diff
@@ -32,6 +32,8 @@ psycopg2-binary = "^2.9.9"
 uvicorn = {extras = ["standard"], version = "^0.29.0"}
 playwright = "^1.44.0"
 pytest-playwright = "^0.5.0"
+pyyaml = "^6.0.1"
+isort = "^5.13.2"
```

Comment:
What is isort needed for (ruff does this for us, right?)

Reply:
yes that is right, Christopher called me back on this, but then i didn't remove it will do